<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar,QMenuBar::item{
	color: rgb(255, 255, 255);
	background-color: rgb(79, 210, 223);
}
QMenu{
color: rgb(255, 255, 255);
background-color: rgb(94, 198, 204);

}
QMenuBar{
	border-width: 1px;
    border-style: none none solid none;
    border-color: rgb(158, 158, 158);
}

QMenu{
	border-width: 2px;
    border-style: inherit solid solid inherit;
    border-color: rgb(158, 158, 158);
}
QMenuBar::item:selected ,QMenu::item:selected{ 
	color: rgb(0, 0, 0);
    background-color:rgb(255, 255, 255);
}
QDockWidget::title {
    text-align: central;
}
</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
	gridline-color: rgb(213, 17, 17);
}</string>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionSet"/>
    <addaction name="act_qiangzhi"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_F11"/>
    <addaction name="action_s"/>
    <addaction name="actionC"/>
    <addaction name="actionB"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action_mulu"/>
    <addaction name="action_shuqian"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_2"/>
   <addaction name="menuSettings"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSet">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>显示设置</string>
   </property>
   <property name="toolTip">
    <string>显示设置</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="act_qiangzhi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>强制执行“高级设置”</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_books">
   <property name="text">
    <string>首页</string>
   </property>
   <property name="toolTip">
    <string>最近阅读过的书籍</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_F11">
   <property name="text">
    <string>F11 - 全屏切换</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_s">
   <property name="text">
    <string>s - 设置</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>c - 目录</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_mulu">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>目录</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_shuqian">
   <property name="text">
    <string>书签</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>添加书签</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>b - 书签</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>← - 上一页</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>→ - 下一页</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
